artemis-jms/src/test/resources/broker.xml

```diff
@@ -0,0 +1,23 @@
+<configuration xmlns="urn:activemq" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:activemq /schema/artemis-configuration.xsd">
+    <core xmlns="urn:activemq:core">
+        <paging-directory>./target/artemis/paging</paging-directory>
+        <bindings-directory>./target/artemis/bindings</bindings-directory>
+        <journal-directory>./target/artemis/journal</journal-directory>
+        <large-messages-directory>./target/artemis/large-messages</large-messages-directory>
+
+        <connectors>
```

Comment:
More a proposal for the extension itself.
Would it be possible to configure the extension from the `application.properties` file?

Reply:
This is the server configuration. The extension can only configure the client aspects. As far as I know, large part of the configuration is only configurable in the server.

Reply:
Oh sorry, my bad!